musicbot/scrobble.py

```diff
@@ -0,0 +1,78 @@
+import urllib
+import json
+import hashlib
+import time
+import sys
+
+sk = ''
+api_key = ''
+api_secret = ''
+
+class Scrobble(object):
+	def __init__(self, user, passwd, key, secret):
```

Comment:
In Python you can't mix tabs and spaces and it's not recommended to do so inter-file mixing like this. Stick to spaces and don't use tabs.